django/contrib/postgres/aggregates/general.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models.aggregates import Aggregate
+
+__all__ = [
+    'ArrayAgg', 'BitAnd', 'BitOr', 'BoolAnd', 'BoolOr', 'StringAgg',
+]
+
+
+class ArrayAgg(Aggregate):
+    function = 'ARRAY_AGG'
+
+    def convert_value(self, value, expression, connection, context):
+        if not value:
+            return []
+        return value
+
+
+class BitAnd(Aggregate):
+    function = 'BIT_AND'
+
+
+class BitOr(Aggregate):
+    function = 'BIT_OR'
+
+
+class BoolAnd(Aggregate):
+    function = 'BOOL_AND'
+
+
+class BoolOr(Aggregate):
+    function = 'BOOL_OR'
+
+
+class StringAgg(Aggregate):
+    function = 'STRING_AGG'
+    template = "%(function)s(%(expressions)s, '%(delimiter)s')"
+
+    def __init__(self, *expressions, **extra):
```

Comment:
I think the definition for this might be clearer as:

```
def __init__(self, expression, delimiter, **extra):
    super(StringAgg, self).__init__(expression, delimiter=delimiter, **extra)
```

The docs already have this definition, and multiple expressions do not make sense for this aggregate. You can also ditch the kwarg processing to check for delimiter (which is inconsistent with the docs at this point).